src/libaktualizr-posix/ipuptanesecondary.h

```diff
@@ -7,22 +7,39 @@
 #include "uptane/secondaryinterface.h"
 
 namespace Uptane {
+
 class IpUptaneSecondary : public SecondaryInterface {
  public:
-  explicit IpUptaneSecondary(const SecondaryConfig& conf) : SecondaryInterface(conf) {}
+  static std::shared_ptr<Uptane::SecondaryInterface> create(const std::string& address, unsigned short port);
+
+  explicit IpUptaneSecondary(const sockaddr_in& sock_addr, EcuSerial serial, HardwareIdentifier hw_id,
+                             PublicKey pub_key);
+
+  // what this method for ? Looks like should be removed out of SecondaryInterface
```

Comment:
It's used by `ManagedSecondary`. It's possible that it could be merged into the constructor, but I think there used to be a good reason it was necessary before. It may not be necessary now. Not priority, though. We can look into that further once more of the refactoring is done.

Reply:
ok.